sdks/python/apache_beam/transforms/ptransform.py

```diff
@@ -639,6 +646,8 @@ def default_label(self):
 def ptransform_fn(fn):
   """A decorator for a function-based PTransform.
 
+  For internal use only; no backwards-compatibility guarantees.
```

Comment:
Not for internal use, just note that it's expiremental (and, yes, not backwards compatible).